### DISCUSSION TITLE
Add beat snap grid to taiko editor

### DISCUSSION THREAD

**Discussion by durshlakk** [ppy/osu#22177] [Ideas] [answered]
Hi!
For the osu!taiko editor can make strips of ticks on the field itself, so that it is convenient to put kats and dons
![taiko](https://user-images.githubusercontent.com/77969719/212339757-f5077780-3763-4b0d-b0d0-1831f2b1a05b.png)

Also in osu!stable it is customary to delete notes with the right mouse button. And to do it the way it is now in osu!!lazer, the menu opens when you click on a note, you need a Shift + right mouse button combination

**Comment by bdach** [comment]
Quick delete in lazer is shift+right click right now.

The snap grid could be added, for sure.

**Comment by TheFearlessDeath500** [comment]
Wouldn’t this be messy in hindsight for new mappers just starting out? If this were to be implemented, barlines and snap grid lines should be clearly shown.

**Reply by Rekunan** [reply]
I don't see how this would be messy, if anything it'd be more helpful. Also, the example image has the lines clearly shown by color coding.

**Comment by Hiviexd** [comment, marked as answer]
https://github.com/ppy/osu/pull/25154
